content/docs/fundamentals/converting/resolving-errors@id.md

```diff
@@ -0,0 +1,350 @@
+{% endcall %}
+
+Coba buka file [`base.js`](https://github.com/googlecodelabs/accelerated-mobile-pages-foundations/blob/master/base.js) eksternal. Apa yang Anda lihat? File tidak boleh berisi kode JavaScript sama sekali dan hanya boleh menyertakan komentar informasi sebagaimana berikut:
+
+```javascript
+/*
+
+File JavaScript eksternal ini sengaja dikosongkan.
+
+Tujuannya hanyalah untuk menunjukkan error validasi AMP yang terkait dengan
+penggunaan file JavaScript eksternal.
+
+*/
+```
+
+Karena file JavaScript eksternal ini bukan komponen fungsional pada situs, kita dapat menghapus referensi sepenuhnya.
+
+**Hapus** referensi JavaScript eksternal berikut dari dokumen Anda:
+
+```html
+<script type="text/javascript" src="base.js"></script>
+```
+
+Sekarang, **muat ulang** halaman dan verifikasi bahwa error skrip sudah tidak muncul.
+
+## Sertakan boilerplate CSS AMP
+
+Error berikut merujuk pada kode boilerplate yang tidak ada:
+
+<pre class="error-text">
+Tag wajib 'noscript enclosure for boilerplate' tidak ada atau salah.
+Tag wajib 'head > style : boilerplate' tidak ada atau salah.
+Tag wajib 'noscript > style : boilerplate' tidak ada atau salah.
+</pre>
+
+Setiap dokumen AMP memerlukan kode boilerplate AMP berikut:
+
+```html
+<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+```
+
+**Tambahkan** kode boilerplate ke bagian bawah tag `<head>` pada dokumen Anda.
+
+Tag `<style amp-boilerplate>`  awalnya menyembunyikan konten di bagian isi sampai library JavaScript AMP dimuat, kemudian konten tersebut akan dirender. AMP melakukannya untuk mencegah agar konten yang tidak bergaya tidak render, juga dikenal sebagai Flash Of Unstyled Content (FOUC). Ini membantu memastikan bahwa pengalaman pengguna terasa benar-benar instan karena konten halaman muncul sekaligus dan semua yang ada di paruh atas ditampilkan secara bersamaan. Tag kedua akan mengalihkan logika ini jika JavaScript dinonaktifkan di browser.
+
+## ganti `<img>` dengan `<amp-img>`
+
+AMP tidak mendukung partner HTML default untuk menampilkan media, yang menjelaskan error berikut:
+
+<pre class="error-text">
+Tag 'img' mungkin hanya muncul sebagai turunan tag 'noscript'. Apakah maksud Anda 'amp-img'?
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).

Reply:
Updated code block with EN version.